sonaranalyzer-dotnet/rspec/cs/S3966_c#.html

```diff
@@ -1,33 +1,20 @@
 <p>A proper implementation of <code>IDisposable.Dispose</code> should allow for it to be called multiple times on the same object, however this is not
 guaranteed and could result in an exception being thrown.</p>
-<p>It is best not to rely on this behaviour and therefore make sure an object is disposed only once on all execution paths. This is particularly true
+<p>It is best not to rely on this behavior and therefore make sure an object is disposed only once on all execution paths. This is particularly true
```

Comment:
is the nested using statements still valid?
> This is particularly true when dealing with nested using statements.


I would check with @nicolas-harraudeau-sonarsource on how it's best to word the new implementation

Reply:
Thanks for pinging me @andrei-epure-sonarsource.
I rewrote the rule description with the help of @costin-zaharia-sonarsource.

Reply:
@andrei-epure-sonarsource @costin-zaharia-sonarsource I'm just wondering if we shouldn't keep the rule in sonar-way now that we removed the problematic use case. disposing the object twice explicitly is really strange to me.

Reply:
After discussion with @nicolas-harraudeau-sonarsource we decided to add the rule back to Sonar Way. With the latest changes most of the FPs will disappear and the found issues on peach reveal code which is error prone.